src/coreclr/scripts/superpmi.py

```diff
@@ -246,6 +244,75 @@ def safe_char(c):
             return "_"
     return "".join(safe_char(c) for c in s)
 
+def _find(name, pathlist, matchFunc=os.path.isfile):
+    """ Find a name (e.g., directory name or file name) in the file system by searching the directories
+        in a semicolon-separated `pathlist` (e.g., PATH environment variable).
+
+    Args:
+        name (str)              : name to search for
+        pathlist (str)          : semicolon-separated string of directory names to search
+        matchFunc (str -> bool) : determines if the name is a match
+
+    Returns:
+        (str) The pathname of the object, or None if not found.
+    """
+    for dirname in pathlist:
+        candidate = os.path.join(dirname, name)
+        if matchFunc(candidate):
+            return candidate
+    return None
+
+def findFile(filename, pathlist):
```

Comment:
could you please rename it to `find_file` while you are here? Looks like that file mostly uses snake_case.